assets/src/domain/eventEditor/ui/tickets/hooks/useTicketsRegistrationCount.ts

```diff
@@ -1,17 +0,0 @@
-import { EntityId } from '@appServices/apollo/types';
-
-/**
- * A custom hook that holds the registration count of all the tickets.
- */
-const useTicketsRegistrationCount = () => {
-	// @todo get registration count
-	const allTicketsRegistrationsCount = {}; // {ticket.id: RegCount}
-
-	const getRegCount = (ticketId: EntityId): number => {
-		return allTicketsRegistrationsCount[ticketId] || 0;
-	};
-
-	return { getRegCount };
-};
```

Comment:
nice that you removed this, it was a bit confusing

Reply:
Agreed